lib/server.js

```diff
@@ -119,5 +119,4 @@ module.exports = async config => {
 
 process.on('unhandledRejection', err => {
     logger.error('Unhandled error: %s', err.stack ? err.stack : err);
-    process.exit(1);
```

Comment:
I think it make sense to exit when unhandled error is happened because something may be wrong.
Is it better to handle unhandled errors correctly, isn't it?